tools/ZeroC.Ice.Slice.Tools/TaskUtil.cs

```diff
@@ -5,15 +5,11 @@
 using System.IO;
 using System.Runtime.InteropServices;
 
-namespace Ice.Slice.Tools;
+namespace ZeroC.Ice.Slice.Tools;
 
 public class TaskUtil
 {
-#if NETSTANDARD2_0
```

Comment:
Modern .NET Frameworks support netstandard2.0, don't need this anymore. The MSBuild used by VC++ is still based on .NET Framework.